tests/yam/validation/full_disk_encryption.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+import { StoragePage } from './pages/storage-page';
+import { mainPagePath } from "../../../lib/installer";
+
+test.describe('The main page', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(mainPagePath());
+  });
+
+  test('full disc encryption', async ({ page }) => {
+    await page.getByRole('link', { name: 'Storage' }).click();
+
+    const storage = new StoragePage(page);
+    await storage.enable_encryption();
+    await storage.set_encryption_password('nots3cr3t');
+    await storage.checkEncryptionEnabled();
+    await storage.exitStorageSettings();
```

Comment:
it is more like `storagePage.backToMainPage()`;

Reply:
Then you need to add the non-pom auto-generated code for adding user / root info and install, confirm install, etc. but instead of using the auto-generated you can borrow it from one of the test that already does that, well actually one part is in one ci test and the install part in the take_screenshots, so you can just borrow code from those.